tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py

```diff
@@ -32,36 +32,45 @@ def setUp(self):
                              }
         self.CLEANUP["accounts"].append(self.acc1)
 
-    @unittest.skip('https://github.com/openvcloud/0-templates/issues/47')
+    def tearDown(self):
```

Comment:
we delete account on utils which delete cloudspaces inside it too, so why we add teardown here too ?

Reply:
now you need to clean the vms before you delete the cloudspace and u need to delete vdcs before u delete account.. other than this u will get an error saying "cloudspace or account is not empty"